docs/rules/configurable-rules.md

```diff
@@ -65,6 +65,8 @@ A minimum of one assertion property is required to be defined.
 ## Where object
 
 The `where` object is part of a `where` list which must be defined in order from the root node.
+A node cannot be used in the `wrhere` list twice.
+However, the same node could be used in the latest `where` object and in the root `subject` object.
```

Comment:
I wonder if "latest" is clear? is that first? last?

Reply:
I meant to write last, thanks.